fe1-web/res/strings.ts

```diff
@@ -60,6 +60,7 @@ const STRINGS = {
   /* --- Social Media Strings --- */
   button_publish: 'Publish',
   your_chirp: 'Your chirp',
+  deleted_chirp: 'This chirp was deleted',
```

Comment:
```suggestion
  deleted_chirp: 'This chirp has been deleted',
```

Reply:
I will keep the 'was' as this is the word used in Slack.